integrationtest/mapper/orm-spring-uberjar/test/pom.xml

```diff
@@ -87,14 +87,14 @@
                     <execution>
                         <phase>verify</phase>
                         <goals>
-                            <goal>java</goal>
+                            <goal>exec</goal>
                         </goals>
                         <configuration>
                             <!--
                                 We don't want to execute the spring jar if we are not running the tests.
                              -->
                             <skip>${failsafe.spring.skip}</skip>
-                            <mainClass>org.springframework.boot.loader.JarLauncher</mainClass>
+                            <executable>${java-version.test.launcher}</executable>
```

Comment:
We might want this on 6.2 as well, be it only for consistency? Just so we don't get surprised if we end up enabling Spring Boot 3 + JDK 21 tests on that branch one day.